client/src/screens/burger/Root.js

```diff
@@ -121,8 +124,20 @@ class BurgerScreen extends Component {
     }
   };
 
-  sendTestPush = () => {
-    this.props.sendTestPush().catch((e) => {
+TestPushPrompt = () => {
```

Comment:
this is another example of where eslint is failing to warn about indentation. Also please don't start a regular instance method with a capital letter.